src/main/java/org/folio/rest/support/BulkProcessingErrorFileWriter.java

```diff
@@ -0,0 +1,91 @@
+package org.folio.rest.support;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.file.AsyncFile;
+import io.vertx.core.file.OpenOptions;
+import io.vertx.core.json.Json;
+import java.nio.file.Path;
+import java.util.function.Function;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.services.BulkProcessingContext;
+
+public class BulkProcessingErrorFileWriter {
```

Comment:
@RuslanLavrov  Let's add documentation for this class for a better understanding of the main purpose of this entity.

Reply:
added